### PR TITLE
add some small bits of code missing from this repo, add Prop util

### PR DESCRIPTION
**Pull request by hotaru355**
## Problem

- There are some slight discrepancies between the version in the helio repo and this open source version
- There is a small util I have lying around, that is totally optional to use and I would to see if it is useful

https://dronebase.atlassian.net/browse/DAN-273

## Changes

- added Kaymans fix to the `Include` error messaging
- added a `Prop` util class

### CODE REVIEW

File: cuckoo/utils.py [hunk 1]
```diff
@@ -69,3 +69,174 @@ def grouper(
         tuple(entry for entry in iterable if entry is not None)
         for iterable in zip_longest(*iterables, fillvalue=None)
     )
+
+
+class Prop:
```
Comment (hotaru355) [inline cuckoo/utils.py:74]: I have decided to use an `_` _at the end_ of the method name. reasoning:
- `and`, `or`, `with` are reserved words and cannot be used as method names
- `_` at the beginning of the method indicates a private method
- I have added the `_` to all methods to be consistent

Let me know if you have ideas for better naming. thanks

**Review by hotaru355 — COMMENTED** [review]
review comments

File: cuckoo/utils.py (hunk 1 above)
Comment (kaymanb) [reply cuckoo/utils.py:74]: Hmm, interested to try this out. Personally, I think typing the raw dictionary is fine, but I can see some benefit here, with things like `Prop("size_dc_mw").asc_()`.

**Review by kaymanb — APPROVED** [review]

File: cuckoo/utils.py (hunk 1 above)
Comment (clvcooke) [reply cuckoo/utils.py:74]: Yeah I am unlikely to use this right away, but maybe once I see it in action it'll be convinced

**Review by clvcooke — APPROVED** [review]

**Pull request merged**